Makefile

```diff
@@ -9,6 +9,7 @@ help:
 WARNINGS=RUSTDOCFLAGS="-D warnings"
 DEBUG_ASSERTIONS=RUSTFLAGS="-C debug-assertions"
 ALL_FEATURES_BUT_ASYNC=--features concurrent,testing
+BUILD_PROTO=BUILD_PROTO=1
```

Comment:
Related to one of the previous comments: I think we can avoid this `BUILD_PROTO` variable.